spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkCatalog.java

```diff
@@ -31,8 +31,10 @@
 import org.apache.spark.sql.connector.catalog.SupportsNamespaces;
 import org.apache.spark.sql.connector.catalog.Table;
 import org.apache.spark.sql.connector.catalog.TableCatalog;
+import org.apache.spark.sql.connector.catalog.ViewCatalog;
 
-public class TestSparkCatalog<T extends TableCatalog & FunctionCatalog & SupportsNamespaces>
+public class TestSparkCatalog<
+        T extends TableCatalog & FunctionCatalog & SupportsNamespaces & ViewCatalog>
```

Comment:
Do we also need to rewrite function names in views?

Reply:
at this point I don't know actually

Reply:
Can you do some checking to see how function names are resolved? I think they at least come from the current catalog. We just need to check whether we should rewrite them.